.vsconfig

```diff
@@ -50,7 +50,7 @@
     "Microsoft.VisualStudio.Component.VC.ASAN",
     "Microsoft.VisualStudio.Component.Vcpkg",
     "Microsoft.VisualStudio.Component.Windows10SDK.19041",
```

Comment:
Do we need to keep 19041?

Reply:
Probably not, I don't see anything using it in this solution.  I can remove it in another PR to prevent the need to rebuild this one.